doc/api/dns.md

```diff
@@ -568,6 +568,425 @@ An error will be thrown if an invalid address is provided.
 The `dns.setServers()` method must not be called while a DNS query is in
 progress.
 
+## DNS Promises API
+
+> Stability: 1 - Experimental
+
+The `dns.promises` API provides an alternative set of asynchronous DNS methods
+that return `Promise` objects rather than using callbacks. The API is accessible
+via `require('dns').promises`.
+
+## Class: dnsPromises.Resolver
+<!-- YAML
+added: REPLACEME
+-->
+
+An independent resolver for DNS requests.
+
+Note that creating a new resolver uses the default server settings. Setting
+the servers used for a resolver using
+[`resolver.setServers()`][`dns.setServers()`] does not affect
+other resolvers:
+
+```js
+const { Resolver } = require('dns').promises;
+const resolver = new Resolver();
+resolver.setServers(['4.4.4.4']);
+
+// This request will use the server at 4.4.4.4, independent of global settings.
+resolver.resolve4('example.org').then((addresses) => {
+  // ...
+});
+```
+
+The following methods from the `dnsPromises` API are available:
+
+* [`resolver.getServers()`][`dnsPromises.getServers()`]
+* [`resolver.setServers()`][`dnsPromises.setServers()`]
```

Comment:
We have `dnsPromises.getServers()` section, but no `dnsPromises.setServers()` one.